docs/examples/event-processing.md

```diff
@@ -0,0 +1,211 @@
+
+```python doctest:SKIP
+import datetime
+from collections import defaultdict
+
+from bytewax.window import TumblingWindowConfig, SystemClockConfig
+
+cc = SystemClockConfig()
+wc = TumblingWindowConfig(length=datetime.timedelta(seconds=5))
+
+def build():
+    return defaultdict(lambda: 0)
+
+def count_events(results, event):
+    results[event["type"]] += 1
+    return results
+
+flow.fold_window("session_state_recovery", cc, wc, build, count_events)
+```
+
+Output mechanisms in Bytewax are managed in the [capture operator](https://docs.bytewax.io/apidocs/bytewax.dataflow#bytewax.dataflow.Dataflow.capture). There are a number of helpers that allow you to easily connect and write to other systems ([output docs](https://docs.bytewax.io/apidocs/bytewax.outputs)). If there isn’t a helper built, it is easy to build a custom version. Like the input, Bytewax output can be parallelized and connection will occur on the worker.
+
+```python doctest:SKIP
+import json
+
+import psycopg2
+from bytewax.outputs import ManualOutputConfig
+
+def output_builder(worker_index, worker_count):
+    # create the connection at the worker level
+    conn = psycopg2.connect("dbname=website user=bytewax")
+    conn.set_session(autocommit=True)
+    cur = conn.cursor()
+
+    def write_to_postgres(user_id__user_data):
+        user_id, user_data = user_id__user_data
+        query_string = '''
+                    INSERT INTO events (user_id, data)
+                    VALUES (%s, %s)
+                    ON CONFLICT (user_id)
+                    DO
+                        UPDATE SET data = %s;'''
+        cur.execute(query_string, (user_id, json.dumps(user_data), json.dumps(user_data)))
+    return write_to_postgres
+
+flow.capture(ManualOutputConfig(output_builder))
+```
+
+Bytewax comes with a few different execution models. They are used to run the dataflow in different manners, like running across a cluster or running on a local machine. Below is an example of running on across a manual managed cluster
+
+```python doctest:SKIP
+if __name__ == "__main__":
+    addresses = [
+    "localhost:2101"
+    ]
+
+    cluster_main(
+        flow,
+        addresses=addresses,
+        proc_id=0,
+        worker_count_per_proc=2)
+```
+
+**Deploying and Scaling**
+--------
+
+Bytewax can be run like a regular python script using a single worker thread and process. Alternatively it could be scaled up to multiple worker threads and processes.
```

Comment:
```suggestion
Bytewax can be run like a regular Python script using a single worker thread and process. Alternatively, it could be scaled up to multiple worker threads and processes.
```
Sometimes it's capitalized here 🤷